### DISCUSSION TITLE
Markdown works in projects, not in blog posts

### DISCUSSION THREAD

**Discussion by JohannesAck** [alshedivat/al-folio#641] [Q&A] [answered]
I really like this theme and it's great for building a personal website, thanks! I have a small issue with blog posts:

I would like to use the Kramdown [markdown syntax](https://kramdown.gettalong.org/quickref.html) in blog posts, for example for footnotes and also for convenience as with links etc.

However, while the markdown support works great for project posts, it doesn't seem to work for blog posts.
I was wondering if there are some special changes needed for it to work. I tried changing the layout from `post` to `page`, but it seems like this didn't change anything.

Thanks a lot for your help!

**Comment by cmoralesmx** [comment, marked as answer]
Hi, would you mind sharing an example of what is not working for you?  

In my own site, for blog posts, I use the same markdown style I use here for creating links, for embedding code, and so on, all with no issue.  

Here's an example from one of the stock blog post entries using markdown to create a link
https://github.com/alshedivat/al-folio/blob/30512eebda1fb9dd46bb107e869d5ec8de9b9411/_posts/2020-09-28-twitter.md?plain=1#L20

Also code highlighting works with the stock settings, as shown in this tiny example
![image](https://user-images.githubusercontent.com/1790200/194723137-d550e0b7-f1c4-4106-a52c-99d777c7d880.png)
which was produced by adding the following to a blog post entry,

````
Markdown style:
```python
import pandas as pd

numbers = []
for i in numbers:
    echo i
```
````

Perhaps you are trying to do more advance formatting. Have you checked the markdown section in the Jekyll's [docs](https://jekyllrb.com/docs/configuration/markdown/)?

**Reply by JohannesAck** [reply]
Hey, sorry I never got back to this. I think something was wrong in my config repo, after resetting it it works perfectly now.

Thanks a lot!
